features/kinderen.feature

```diff
@@ -121,7 +133,7 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 000000073                       |
       | fields              | kinderen.burgerservicenummer    |
-	  Dan heeft de response een persoon zonder 'kind' gegevens
+	  Dan heeft de response een persoon met een 'kind' zonder gegevens
```

Comment:
Volgens het gba-dev scenario levert dit een response met een persoon zonder gegevens op en geen kind. Dat is volgens mij ook de correcte respons.

Reply:
Nee, volgens mij is het gerelateerde gba scenario incorrect. Er is immers wel een kind en dat moet getoond worden. De gevraagde gegevens van dat kind zijn echter niet beschikbaar dus moet er een leeg 'kind' geleverd worden. Als dat gebeurd dan test dit scenario ook correct omdat de juiste response wordt geleverd aan de proxy.

Reply:
De conclusie is dat de gba-dev scenario inderdaad incorrect is.